### PR TITLE
[zh] Sync cluster-administration pages for addons.md

### PR DESCRIPTION
**Pull request by ydFu**
* Sync with english version in 'Fix Contiv addon link.' (#32938)

Signed-off-by: ydFu <ader.ydfu@gmail.com>

### CODE REVIEW

File: content/zh/docs/concepts/cluster-administration/addons.md [hunk 1]
```diff
@@ -54,6 +55,7 @@ Add-ons 扩展了 Kubernetes 的功能。
   同时支持路由（routing）和覆盖/封装（overlay/encapsulation）模式。
 * [CNI-Genie](https://github.com/Huawei-PaaS/CNI-Genie) 使 Kubernetes 无缝连接到一种 CNI 插件，
   例如：Flannel、Calico、Canal、Romana 或者 Weave。
+* [Contiv](https://contivpp.io/) 为各种用例和丰富的策略框架提供可配置的网络（使用 BGP 的本机 L3、使用 vxlan 的覆盖、标准 L2 和 Cisco-SDN/ACI）。 Contiv 项目完全 [开源](https://github.com/contiv)。 [安装程序](https://github.com/contiv/install) 提供了基于 kubeadm 和非 kubeadm 的安装选项。
```
Comment (tengqm) [inline content/zh/docs/concepts/cluster-administration/addons.md:58]:
```suggestion
* [Contiv](https://contivpp.io/) 为各种用例和丰富的策略框架提供可配置的网络
  （使用 BGP 的本机 L3、使用 vxlan 的覆盖、标准 L2 和 Cisco-SDN/ACI）。
  Contiv 项目完全[开源](https://github.com/contiv)。
  [安装程序](https://github.com/contiv/install) 提供了基于 kubeadm 和非 kubeadm 的安装选项。
```

**Comment by ydFu** [comment]
/assign @tengqm 
Thank for your code review.

**Pull request merged**